src/Entities/Regular/RegularSingleton.java

```diff
@@ -10,29 +10,23 @@
     String id;
     String name;
     String description;
-    float price;
-    List<IAddon> add_ons;
-    String ID;
+    List<IAddon> addons;
     ISelection defaultSelection;
 
     /**
-     * Construct an instance of a RegularFood, which is an object sold by a food truck vendor.
+     * Creates a RegularSingleton object
      *
-     * @param id          id of the item
-     * @param name        Name of the item
-     * @param price       Price of the item
```

Comment:
Singletons should have a price as this will be used as the default for calculating the food price

Reply:
Yup, I will put it back